games-simulation/openrct2/openrct2-0.1.2.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils
+
+DESCRIPTION="An open source re-implementation of RollerCoaster Tycoon 2"
+HOMEPAGE="https://openrct2.website/"
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/OpenRCT2/OpenRCT2.git"
+	EGIT_BRANCH="develop"
+	inherit git-r3
+	SRC_URI=""
+else
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="https://github.com/OpenRCT2/OpenRCT2/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/OpenRCT2-${PV}"
+fi
+
+TSV="0.1.2"
```

Comment:
Why not use `${PV}` here? If somebody else bumps it, he may miss bumping this number when upstream releases new TS. On the other hand, if you put `${PV}` here and upstream doesn't release new TS, the ebuild will simply fail and TSV will be updated.